src/pravin/Assignment_15/Bank.java

```diff
@@ -0,0 +1,142 @@
+/*Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2 */
+
+package pravin.Assignment_15;
+	
+	public class Bank {
+		int iBalance=1000;
+		int sBalance=3000;
+		int hBalance=4000;
+		int bBalance=5000;
+		int TotalDebitCount;
+		int TotalCreditCount;
+		int iDebitCount = 0,iCreditCount=0,sDebitCount = 0,sCreditCount=0,hDebitCount = 0,hCreditCount=0,bDebitCount = 0,bCreditCount=0;
+	
+		void operation(String op,String bank,int amt) {
+			switch (op){
+			case "debit":
+				switch (bank) {
+				case "icici":
+					if(iBalance>=amt) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.